utilities/utils/response_windows.go

```diff
@@ -49,3 +51,39 @@ func getIP(inspect types.ContainerJSON, cache *cache.Cache) (string, error) {
 	hijack.Close()
 	return ip, nil
 }
+
+func setupDNS(containerID string) {
+	command := []string{"powershell", "Get-NetAdapter", "|", "Set-DnsClientServerAddress", "-ServerAddresses"}
+	addressesArg := ""
+	for i, ip := range strings.Split(config.DNSAddresses(), ",") {
+		if i > 0 {
+			addressesArg += ","
+		}
+		addressesArg += fmt.Sprintf("'%s'", ip)
+	}
+	command = append(command, fmt.Sprintf("(%s)", addressesArg))
+	createAndStart(containerID, command)
```

Comment:
IMO throwing an error in `createAndStart` would be better because later on we can decide how to deal with the error gracefully in top-level method. @LLParse Not sure what we should do if error pops out and DNS-address injection fails. Should we fail to start the container?